readme-vars.yml

```diff
@@ -0,0 +1,38 @@
+---
+
+# project information
+project_name: xbackbone
+project_url: "https://github.com/SergiX44/XBackBone"
+project_logo: ""
+project_blurb: |
+  [{{ project_name|capitalize }}]({{ project_url }}) is an image hosting software that allows you to create a beautiful and full-featured image hosting website on your own server. It's your hosting and your rules, so say goodbye to closures and restrictions.
+project_lsio_github_repo_url: "https://github.com/linuxserver/docker-{{ project_name }}"
+
+# supported architectures
+available_architectures:
+  - { arch: "{{ arch_x86_64 }}", tag: "amd64-latest"}
+  - { arch: "{{ arch_arm64 }}", tag: "arm64v8-latest"}
+  - { arch: "{{ arch_armhf }}", tag: "arm32v7-latest"}
+
+# container parameters
+param_container_name: "{{ project_name }}"
+param_usage_include_vols: true
+param_volumes:
+  - { vol_path: "/config", vol_host_path: "<path to data on host>", desc: "config directory volume mapping" }
+param_usage_include_env: true
+param_env_vars:
+  - { env_var: "TZ", env_value: "<TZ>", desc: "Timezone (i.e., America/New_York)" }
```

Comment:
Remove <> from the values, provide working examples.

Reply:
done